composer.json

```diff
@@ -19,15 +19,24 @@
         }
     ],
     "require": {
-        "php": "~5.4|~7.0"
+        "php": "^7.3"
     },
     "require-dev": {
-        "phpunit/phpunit":           "~4.0|~5.0",
-        "squizlabs/php_codesniffer": "~2.5"
+        "phpunit/phpunit":           "^8.0.4",
+        "squizlabs/php_codesniffer": "^3.4.0",
+        "doctrine/coding-standard": "^5.0",
+        "phpstan/phpstan": "^0.11.2",
```

Comment:
Not for now, but maybe we can use `phpstan/phpstan-strict-rules`

Reply:
Added an issue for that: https://github.com/Ocramius/LazyProperty/issues/6